src/features/projectsV2/ProjectsMap/TimeTravel/index.tsx

```
@@ -0,0 +1,496 @@
+
+import {
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
+import MaplibreCompare from '@maplibre/maplibre-gl-compare';
+import '@maplibre/maplibre-gl-compare/dist/maplibre-gl-compare.css';
+import { Map } from 'maplibre-gl';
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)